pkgs/tools/security/osv-scanner/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  testers,
+  osv-scanner,
+}:
+buildGoModule rec {
+  pname = "osv-scanner";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-AftJS8IdKc1oDJfDTB/HMLHu7EAVnPGPR61SOlTnC2E=";
```

Comment:
new version, use `hash`:
```suggestion
    hash = "sha256-RmR6ZJg+UkE+eSmz4hGuMlObl6UvnGKNoLtBGVKoQ8Q=";
```

Reply:
Why do you prefer `hash` over `sha256`?

Reply:
I can find a better link if you would like, but basically to allow us to swap out sha256 for the next generation hash, and better align with upstream syntax:
https://discourse.nixos.org/t/is-it-ok-to-ask-for-hash-attribute-and-sri-hash-when-reviewing-prs-that-update-packages/20493

it is not required, just suggested; if you prefer, feel free to leave it as `sha256`, but it will likely change eventually.

Reply:
No worries, thanks for the context.

Reply:
yeah, there is a lot of institutional knowledge packed into the recesses of this repository